docs/commands.md

```diff
@@ -43,6 +43,7 @@ Usage:
   kics scan [flags]
 
 Flags:
+      --add-gitignore                 disables the exclusion of paths specified within .gitignore file
```

Comment:
Hi @rafaela-soares , revisiting this, the flag add-gitignore sounds misleading.
Taking the suggestion from @roi-orca (and considering that the default will be to include gitignore) WDYT if we change to `--exclude-gitignore` ?
If agreed, please make the remainder of the code consistent to that new naming.